thoth/slo_reporter/sli_workflow_quality.py

```diff
@@ -101,37 +101,36 @@ def _report_sli(self, sli: Dict[str, Any]) -> str:
 
         for service in REGISTERED_SERVICES:
             service_metrics = {}
+
             for metric, value in sli.items():
+
                 if service in metric:
                     service_metrics[metric] = value
 
-            number_reports = sli[f"{service}_reports"]
-            number_workflow_failed = sli[f"{service}_workflows_failed"]
-            number_workflow_error = sli[f"{service}_workflows_error"]
+            number_workflows_succeeded = sli[f"{service}_workflows_succeeded"]
+            number_workflows_failed = sli[f"{service}_workflows_failed"]
+            number_workflows_error = sli[f"{service}_workflows_error"]
 
             if "ErrorMetricRetrieval" not in [v for v in service_metrics.values()]:
 
                 total_workflows = (
-                    int(number_reports)
-                    + int(number_workflow_failed)
-                    + int(number_workflow_error)
+                    int(number_workflows_succeeded) + int(number_workflows_failed) + int(number_workflows_error)
                 )
-                if int(number_reports) > 0:
+                if int(number_workflows_succeeded) > 0:
+
                     successfull_percentage = (
```

Comment:
Maybe also rename this to `successful_percentage` :stuck_out_tongue: Not important :see_no_evil:

Reply:
It's already called `successfull_percentage` if you see one line below :)